port/unix/omrsysinfo.c

```diff
@@ -2058,6 +2059,7 @@ omrsysinfo_startup(struct OMRPortLibrary *portLibrary)
 		}
 	}
 	attachedPortLibraries += 1;
+	isRunningInContainer(portLibrary, &PPG_isRunningInContainer);
```

Comment:
Would it make more sense for this code to look like this:
`PPG_isRunningInContainer = isRunningInContainer(portLibrary);`

Reply:
Any update on this question?

Reply:
Sorry for delayed reply @charliegracie I have missed to reply to this question earlier.

`isRunningInContainer` is an internal method which is used by many functions and some of them (now or in future) might need the `return code` of it. So I'm thinking it would be better to have the current implementation to have the error code handy. Please let me know your views on it. Thanks!

Reply:
No problem :). Ah I missed the other callers.  Since the other callers are using the return value I think it makes sense to leave the code as is.  Thanks!